rsync_tmbackup.sh

```diff
@@ -42,7 +42,7 @@ fn_parse_date() {
 }
 
 fn_find_backups() {
-    fn_run_cmd "find "$DEST_FOLDER" -type d -name "????-??-??-??????" -prune | sort -r"
```

Comment:
I took them out, because they break up the outer quotes. (Right?) So you'd end up with multiple strings being concatenated instead of a single string. I don't think we really need them, since we just want to insert the value of `$DEST_DIR` right? That is, there's no need to evaluate $DEST_DIR later on?